modules/cli/test-scripts/streams-tests.sh

```diff
@@ -0,0 +1,250 @@
+            -h | --help )                     usage
+                                            exit
+                                            ;;
+            * )                               error "Unexpected argument $1"
+                                            usage
+                                            exit 1
+        esac
+        shift
+    done
+
+    # Can't really verify that the bootstrap provided is a valid one, but galasactl will pick this up later if not
+    if [[ "${bootstrap}" == "" ]]; then
+        export bootstrap="https://galasa-ecosystem1.galasa.dev/api/bootstrap"
+        info "No bootstrap supplied. Defaulting the --bootstrap to be ${bootstrap}"
+    fi
+
+    info "Running tests against ecosystem bootstrap ${bootstrap}"
+fi
+
+#-----------------------------------------------------------------------------------------                   
+    # Utility functions
+#-----------------------------------------------------------------------------------------                   
+
+function configure_stream {
+    local stream_name=$1
+    local bootstrap=$2
+    
+    local desc_prop_name="test.stream.${stream_name}.description"
+    local maven_prop_name="test.stream.${stream_name}.repo"
+    local testCatalog_prop_name="test.stream.${stream_name}.location"
+    local obr_prop_name="test.stream.${stream_name}.obr"
+
+    # Define property settings with their commands
+    local cmd1="${BINARY_LOCATION} properties set --namespace framework \
+    --name $desc_prop_name \
+    --value Galasa \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd2="${BINARY_LOCATION} properties set --namespace framework \
+    --name $maven_prop_name \
+    --value http://points-to-my-maven-repo.example.org \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd3="${BINARY_LOCATION} properties set --namespace framework \
+    --name $testCatalog_prop_name \
+    --value http://points-to-my-test-catalog.example.org \
+    --bootstrap $bootstrap \
+    --log -"
+
+    local cmd4="${BINARY_LOCATION} properties set --namespace framework \
+    --name $obr_prop_name \
+    --value mvn:myorg/myartifact/0.0.1/obr \
+    --bootstrap $bootstrap \
+    --log -"
+
+    info "Command is: $cmd1"
+    info "Command is: $cmd2"
+    info "Command is: $cmd3"
+    info "Command is: $cmd4"
+
+    $cmd1
+    $cmd2
+    $cmd3
+    $cmd4
+    local rc=$?
+
+    # We expect a return code of 0 because this is a properly formed properties set command.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to create property with name and value used."
+        return 1
+    fi
+    
+    return 0
+}
+
+#-----------------------------------------------------------------------------------------
+# Tests
+#----------------------------------------------------------------------------------------- 
+function streams_create {
+    h2 "Performing properties set with name and value parameter used: create..."
```

Comment:
This message could be clarified to say that we're creating a test stream for these tests?